pkg/server/commands/listusers/list_users_rpc_test.go

```diff
@@ -1312,14 +1309,14 @@ func TestListUsersExclusion(t *testing.T) {
 		  
 		  type group
 			relations
-			  define member: [user, group#member] but not other
-			  define other: [user, group#member]`,
+			  define member: [user, group#member] but not blocked
+			  define blocked: [user, group#member]`,
 
 			tuples: []*openfgav1.TupleKey{
-				tuple.NewTupleKey("group:1", "other", "group:1#member"),
+				tuple.NewTupleKey("group:1", "blocked", "group:1#member"),
 				tuple.NewTupleKey("group:1", "member", "user:will"),
 			},
-			expectedUsers: []string{},
+			expectedUsers: []string{"user:will"},
```

Comment:
This is an incorrect assertion. This is the non-stratifiable example.

If `group:1#member` is blocked then no terminal subject of `group:1#member` could simultaneously be a `member` and not blocked. The outcome here should be an empty list.

Reply:
Correct. This will be addressed in a future PR.